_config/mailcapture.yml

```diff
@@ -6,8 +6,8 @@ Injector:
     class: CaptureMailer
 #    properties:
 #      outboundMailer: %$MailerObject
-#      #Set to false to enable pass through of emails without logging
-#      captureEmails: TRUE
+#      # Set to false to enable pass through of emails without logging
+#      captureEmails: FALSE
```

Comment:
OH right - I was reading the changeblock block from the readme.md above which _isn't_ commented out, and looking at the filename label here, where it is commented, so no biggy there. But yes - if it could be set to the expected default if you copy/paste the config that'd be good.